packages/web-components/src/utils/utils.spec.ts

```diff
@@ -14,6 +14,112 @@ describe('format', () => {
   });
 
   it('formats first, middle and last names', () => {
-    expect(format('Joseph', 'Quincy', 'Publique')).toEqual('Joseph Quincy Publique');
+    expect(format('Joseph', 'Quincy', 'Publique')).toEqual(
+      'Joseph Quincy Publique',
+    );
+  });
+});
+
+describe('getSlottedNodes()', () => {
+  it('returns an array of nodes from a custom element if hasShadowDom is false', async () => {
+    // Shadow DOM Custom Element
+    class CustomElement extends window.HTMLElement {
+      constructor() {
+        super();
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `<h6>ABC</h6><div>DEF</div><p>GHI</p>`;
+      }
+    }
+
+    window.customElements.define('custom-element', CustomElement);
+
+    const defaultElement = document.createElement('custom-element');
+    const slottedNodes = getSlottedNodes(defaultElement, null);
+    expect(slottedNodes[2].nodeName).toEqual('P');
+  });
+
+  it('filters nodes from a custom element in the shadow dom based on nodeName', async () => {
+    // Shadow DOM Custom Element
+    class CustomElement extends window.HTMLElement {
+      constructor() {
+        super();
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `<h6>ABC</h6><div>DEF</div><p>GHI</p>`;
+      }
+    }
+
+    window.customElements.define('custom-element', CustomElement);
+
+    const defaultElement = document.createElement('custom-element');
+    const slottedNodes = getSlottedNodes(defaultElement, 'p');
+    expect(slottedNodes[0].nodeName).toEqual('P');
+  });
```

Comment:
It looks like these two tests both result in `hasShadowDom` being false simply because `<slot>` isn't used in the custom element definition. This results in nothing being in the filtered array:

https://github.com/department-of-veterans-affairs/component-library/blob/309940ca635092368f1da82a8329048b8819ebf9/packages/web-components/src/utils/utils.ts#L21-L24

However, the point of using `getSlottedNodes` is when we know that a component definition has `<slot>`. If we want to have a test for `hasShadowDom` being false, I think we should still use a `<slot>` but have a mock that is closer to what we would get from IE11. Maybe we mock `childNodes` to return an empty array on the `shadowRoot`, for example.